src/main/java/seedu/duke/ui/TextUi.java

```diff
@@ -0,0 +1,88 @@
+package seedu.duke.ui;
+
+import java.util.Scanner;
+
+/**
+ * UI class for reading user inputs and printing outputs
+ */
+public class TextUi {
+    private static final Scanner SCANNER = new Scanner(System.in);
+    private static final String SEPARATOR = "-";
+    private static final String INDENTATION_SPACES = "    ";
+
+    /**
+     * Read user's input command and return back the command string.<br/>
+     * This is for usage. <br/>
+     *
+     * @return user input command with leading/dangling whitespace being removed
+     */
+    public static String getCommand() {
+        String inputLine;
+        String userCommand;
```

Comment:
Hmm any particular reason the initialisation of these variables are not in the same line as the declaration?